synphot/models.py

```diff
@@ -185,7 +185,18 @@ class Box1D(_models.Box1D):
     """Same as `astropy.modeling.functional_models.Box1D`, except with
     ``sampleset`` defined.
 
+    Parameters
+    ----------
+    step : float
+        Distance of first and last points w.r.t. bounding box. In default units
+        (nominally Angstrom). Defaults to 0.01
+
     """
+    def __init__(self, step=0.01, *args, **kwargs):
+
+        super(Box1D, self).__init__(*args, **kwargs)
+        self.step = float(step)
```

Comment:
Is an explicit float cast really needed here?

Reply:
Possibly not, but what could it hurt?  I'm happy to remove if you wish.

Reply:
Let's remove it. Unless you want something like `Box1D(..., step='0.1')` to magically work.